src/components/Map/EditableMap/EditableMap.js

```diff
@@ -0,0 +1,117 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import MapGL from 'react-map-gl'
+import { Editor, DrawPolygonMode, EditingMode } from 'react-map-gl-draw'
+import Icon from '../../Icon'
```

Comment:
we like to use the absolute path of 'components/Icon', just in case this component moves